src/Our.Umbraco.Vorto/Web/Controllers/VortoApiController.cs

```diff
@@ -46,6 +46,7 @@ public object GetDataTypeByAlias(string contentType, string contentTypeAlias, st
                 case "member":
                     ct = Services.MemberTypeService.Get(contentTypeAlias);
                     break;
+                case "settings":
```

Comment:
The issue we have is that in the settings section you could be editing a content type or a media type so you need to figure out what type is being edited and redirect accordingly. Maybe do an early check for "settings" section before the switch statement and figure out what type you are editing then set the section variable to "content" or "media" respectively?